src/dict.c

```diff
@@ -1239,12 +1340,14 @@
 #define GETFAIR_NUM_ENTRIES 15
 dictEntry *dictGetFairRandomKey(dict *d) {
     dictEntry *entries[GETFAIR_NUM_ENTRIES];
-    unsigned int count = dictGetSomeKeys(d,entries,GETFAIR_NUM_ENTRIES);
+    unsigned int count = dictGetSomeKeys(d, entries, GETFAIR_NUM_ENTRIES);
```

Comment:
This is a perfect example of why I don't want to allow a computer to have full control over ... my car ... my meals ... my code's format ... or much of anything.  Yes, if it were being used for cryptography it would be bad, but that's not what it is being used for.